frontend-instructor/src/components/addCourse/addCourse.jsx

```
@@ -32,32 +33,33 @@ export default function AddCourse() {
+        setStatus({
+          error: `Failed: ${result.message || 'Unknown error'}`
+        });
+        alert(`Failed to add course: ${result.message || 'Unknown error'}`);
       }
     } catch (error) {
-      console.error("Error adding course:", error);
-      setStatus({ error: "Failed to add course" });
-      alert("Failed to add course: " + (error.message || "Unknown error"));
+      console.error('Error adding course:', error);
+      setStatus({ error: 'Failed to add course' });
+      alert('Failed to add course: ' + (error.message || 'Unknown error'));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)